Modules/_winapi.c

```diff
@@ -1596,6 +1597,9 @@ _winapi_GetLongPathName_impl(PyObject *module, LPCWSTR path)
         PyErr_SetFromWindowsErr(0);
     }
     return result;
+#else
+    return PyUnicode_FromWideChar(path, wcslen(path));
```

Comment:
```suggestion
    return PyUnicode_FromWideChar(path, -1);
```

Does the same thing, but if we ever need/decide to add safer/faster length calculations, it can all happen inside the function.